### ISSUE TITLE
Change logs (release notes)

### ISSUE THREAD

**Issue by leicray** [openvar/variantValidator#527]
**Is your feature request related to a problem?**
VariantValidator undergoes constant development, but changes and improvements are not formally documented in an easily accessible fashion for end users.

**Describe the solution you'd like**
The VariantValidator home page provides links to its component parts but no "change log" is provided for these individual parts. Change logs ought to be provided for VariantValidator and vv_hgvs on GitHub and change logs for VVDb, VVTA and VVSeqRepo ought to be provided in the individual directories on the data server .

**Comment by leicray** [comment]
I am aware of this type of change log, but it's not easily understandable for the typical end user. I was thinking of something more like a plain text file that **summarises** any new features, updates and bug fixes.

As you say, low priority until we can recruit more staff.

**Comment by ifokkema** [comment]
I agree with both of you; it's important (LOVD also has a separate changelog that is less technical), but a lot of work. I go through all PRs when I make an official release to document all changes, but it can be time-consuming.

**Comment by Peter-J-Freeman** [comment]
On the next release (pending) I think I'll got through the auto generated log and annotate it and comment. Have a little time
